### PR TITLE
Add workflow to automate release to PyPI

### PR DESCRIPTION
**Pull request by engeir**
* release-drafter checks pull requests that are new since last version,
  and adds sections in the metadata of the release note based on the
  labels given to the different pull requests
* release does the actual creating of release, builds the project and
  publish to PyPI

### CODE REVIEW

**Comment by engeir** [comment]
Just uploaded a small workflow that should actually do the job with creating automatic releases when new versions are created.

It needs the [token from PyPI](https://github.com/uit-cosmo/cosmoplots/blob/6786598a7fb0cb86592e4b3291b47ee4e5b7f9a2/.github/workflows/release.yml#L57), could you add that in as a GitHub secret @gregordecristoforo?

With the token added you can test if its working by just doing `poetry version patch` (and update `__init__.py`) and push it to github.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          poetry version $version.dev.$(date +%s)
+      - name: Build package
+        run: |
+          poetry build --ansi
+      - name: Publish package on PyPI
+        if: steps.check-version.outputs.tag
+        uses: pypa/gh-action-pypi-publish@v1.5.0
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_TOKEN }}
```
Comment (gregordecristoforo) [inline .github/workflows/release.yml:57]: change to secrets.PYPI_PASSWORD

**Review by gregordecristoforo — CHANGES_REQUESTED** [review]
Sorry, I didnt name the token PYPI_TOKEN but PYPI_PASSWORD

File: .github/workflows/release.yml (hunk 1 above)
Comment (engeir) [reply .github/workflows/release.yml]: Fixed in 2b4c9f2e403e063f2eadb4d389d5ed05fe0016ca

**Review by gregordecristoforo — APPROVED** [review]

**Pull request merged**